src/components/ElectionJudgeList.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+
+class ElectionJudgeList extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			results: [],
+		};
+		
+		this.displayList = this.displayList.bind(this);
+		this.tableSort = this.tableSort.bind(this);
+	}
+
+	componentWillReceiveProps(nextProps) {
+		this.setState({ results: nextProps.results });
+	}
+
+	displayList() {
+		if (!this.state.results.length) {
+			return (
+				<tr>
+					<td colSpan="5"><p>There are no results to display.</p></td>
+				</tr>
+			);
+		}
+
+		return this.state.results.map((result, index) => {
+			return (
+				<tr key={`${result.PhoneNumber}--${index}`}>
```

Comment:
There is a concern with using index with the phone number, https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

I also think phone number should be fine on it's own.